### PR TITLE
Add Juejin icon

### PR DESCRIPTION
**Pull request by LitoMore**
CleanShot 2023-03-24 at 03 01 50@2x

**Similarweb rank:** [3,790](https://www.similarweb.com/website/juejin.cn/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Juejin is a UGC for global Chinese developers.

### Resources

See the bottom place of https://xitu.juejin.cn. You will find a link for `Press Kit`.

I used the `Press Kit/svg/logo_blue.svg` for processing this icon.

Its primary color is `#007FFF`. You can find the color inside the SVG file.

**Update:**\
Their press-kit site is down somehow, here is the press kit: [juejin-press-kit.zip](https://github.com/simple-icons/simple-icons/files/11526304/juejin-press-kit.zip)

### Difference preview

CleanShot 2023-03-24 at 02 58 49@2x

### CODE REVIEW

**Comment by LitoMore** [comment]
@simple-icons/icon-review Can I get an approval?

**Comment by service-paradis** [comment]
Unfortunately, I'm not able to access https://xitu.juejin.cn. Got a 502 bad gateway response...

**Comment by LitoMore** [comment]
@service-paradis See the description. I've uploaded their press kit ZIP file here.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5934,6 +5934,11 @@
             "hex": "F7DF1E",
             "source": "https://cssinjs.org/"
         },
+        {
+            "title": "Juejin",
+            "hex": "007FFF",
+            "source": "https://xitu.juejin.cn"
```
Comment (adamrusted) [inline _data/simple-icons.json:5940]:
```suggestion
            "source": "https://juejin.cn"
```

There's an SVG source in the header of the site now @LitoMore :)

File: icons/juejin.svg
Comment (adamrusted) [inline icons/juejin.svg:1]: [Slight difference](https://github.com/simple-icons/simple-icons/assets/1830867/194203ef-499a-46d3-945d-927a3bd15688) between proposed new source in red, and this PR in black. Could you take another look?

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by LitoMore** [comment]
image

Updated according to the new source.

**Review by adamrusted — APPROVED** [review]
LGTM, thanks @LitoMore 🎉

**Pull request merged**